apps/nextjs-website/src/lib/fetch/preHeader.ts

```diff
@@ -11,7 +11,6 @@ const PreHeaderButtonCodec = t.strict({
   icon: t.union([MUIButtonIconCodec, t.null]),
   size: MUIButtonSizeCodec,
   variant: t.keyof({
```

Comment:
We may as well remove variant for PreHeaderButton since it's just ignored and always set to 'text'